ctr-nonce-reuse/src/ctr_game.rs

```diff
@@ -134,6 +129,8 @@ impl CtrGame {
     }
 
     /*
+    Dead Code
+    But we can leave it to show how CTR mode is decrypting
```

Comment:
If you want to show how to decrypt it, you can leave the code uncommented and make it `pub` :)
The compiler will check that it's indeed a valid code. So, if you continue developing your demo, you will have no chance of breaking it at some point.

Reply:
hmm, that's because the `ctr_game` module is not public. But you can use the `#[allow(dead_code)]` attribute:
```rust
#[allow(dead_code)]
fn decrypt_bytes(&self, ciphertext_vec: &[u8]) -> Vec<u8> {
    self.encrypt_bytes(ciphertext_vec)
}
```
Just don't forget to comment why this attribute exists and it will be ok :)